flocx_market/matcher.py

```diff
@@ -0,0 +1,107 @@
+import jmespath
+import re
+
+
+def apply_operator(val1, val2, op):
+
+    if val1 is None or val2 is None or op is None:
+        return False
+
+    # null operator
+    if op == 'null':
+        return val1
+
+    all_num_ops = ['==', '!=', '<', '<=', '>', '>=']
+    all_str_ops = ['eq', 'ne', 'startswith', 'endswith', 'matches']
+    all_list_ops = ['in', 'contains']
+
+    # all numeric operations
+    if op in all_num_ops:
+
+        try:
+            val1 = float(val1)
+            val2 = float(val2)
+
+        except ValueError as verr:
+            print (verr.args)
```

Comment:
It doesn't make any sense to call `print` in this code -- it's not going to be running on the console where anyone will see the output. If you need to log a message, using the `logging` module...but in this case, I think you should just remove the `try/except` block; it's okay to raise a `ValueError` if the function receives invalid values (and then add some tests that verify an exception is raised when you pass in invalid data).